### DISCUSSION TITLE
CAPTCHA-like measures in addition to Regulation

### DISCUSSION THREAD

**Discussion by Fluffy728** [authelia/authelia#8522] [Ideas]
Do you agree?

As is from: https://github.com/authelia/authelia/security/advisories/GHSA-cp67-8jp7-9w5x

### Edited by Authelia

To summarize the SA (which was closed by the user after explanation) we already prevent the means by which automatic or manual brute force attacks via [Regulation](https://www.authelia.com/overview/authorization/regulation/) and the relevant [configuration](https://www.authelia.com/configuration/security/regulation/).

The avenue of opening an idea discussion was suggested as a means of users to express their interest in such a feature.

CAPTCHA represents a potentially desirable addition to currently suitably sufficient and secure measures for our users even though it's technically weak and should only be included as an additional measure to slow down brute force attacks (most automated systems can break CAPTCHA-like systems very quickly as noted by various security focused authorities). 

We sincerely appreciate users using the security reporting mechanism in accordance with our [security policy](https://www.authelia.com/policies/security/) even though it did not end up being accepted as a security advisory. We would prefer users use this method to communicate perceived security issues even if it doesn't end up as an advisory rather than risk accidentally publishing security issues publicly before we and users had the opportunity to mitigate this kind of issue.

**Comment by james-d-elliott** [comment]
As there is almost zero interest this is being closed.
